README.md

```diff
@@ -48,10 +48,12 @@ If you do not have a Coreform Cubit license, you may be able to get one through
 Download and extract the PyStell-UW repository:
 
 ```
-git clone https://github.com/aaroncbader/pystell_uw.git
+git clone --single-branch --branch old https://github.com/aaroncbader/pystell_uw.git
 ```
 
-or download the and extract the ZIP file from [PyStell-UW](https://github.com/aaroncbader/pystell_uw). Once extracted, add the repository directory to your `PYTHONPATH`.
+or download the and extract the ZIP file from [PyStell-UW](https://github.com/aaroncbader/pystell_uw/tree/old). Once extracted, add the repository directory to your `PYTHONPATH`.
+
+Note that the `old` branch of PyStell-UW is necessary because there are complications with its `master` branch that prevent its successful integration with ParaStell.
```

Comment:
```suggestion
Note that the `old` branch of PyStell-UW is necessary for compatibility with this version of ParaStell. There are complications with its `master` branch that prevent its successful integration with ParaStell.
```

Reply:
The complications have to do with PyStell-UW itself, rather than ParaStell, but I hope those will be resolved soon (I imagine they will become apparent as soon as testing is developed).

I'll add the verbiage you suggest though!